components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/titleBody/H5TitleBodyView.kt

```diff
@@ -22,21 +22,21 @@ import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import uk.gov.hmrc.components.compose.atom.heading.Heading5
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
 import uk.gov.hmrc.components.compose.ui.theme.hmrc_spacing_16
 
 @Composable
 fun H5TitleBodyView(
-    h5Text: String,
+    titleText: String,
     bodyText: String,
     modifier: Modifier = Modifier,
 ) {
     Row(modifier = modifier) {
         Column {
-            Heading5(
-                text = h5Text,
-            )
+            Heading5(text = titleText)
             Text(
                 text = bodyText,
+                style = HmrcTheme.typography.body,
```

Comment:
`style = HmrcTheme.typography.body`
use as `style =typography.body`
we can remove HmrcTheme i think